python/ql/src/Security/CWE-089/SqlInjection.qhelp

```diff
@@ -9,6 +9,13 @@ If a database query (such as a SQL or NoSQL query) is built from
 user-provided data without sufficient sanitization, a user
 may be able to run malicious database queries.
 </p>
+
+<p>
+This also includes using the <code>TextClause</code> class in the
+<a href="https://pypi.org/project/SQLAlchemy/"><code>SQLAlchemy</code></a> PyPI package,
```

Comment:
Does `<code>` work inside `<a href>`?
I thought I remembered a case were it didn't work.

Reply:
Thanks, I forgot to test it locally. You're right, but it does work if you flip the `<a>` and `<code>` tags :shrug: